src/olympia/constants/permissions.py

```diff
@@ -0,0 +1,57 @@
+from collections import namedtuple
+
+
+AclPermission = namedtuple('AclPermission', 'app, action')
+
+# Special null rule.
+NONE = AclPermission('None', 'None')
+
+# Admin super powers.  Very few users will have this permission (2-3)
+ADMIN = AclPermission('Admin', '%')
+
+# Can view admin tools.
+ADMINTOOLS = AclPermission('AdminTools', 'View')

```

Comment:
I don't think the constant names should be consistent with how they end up in the admin or database. So imho, `ADMIN_TOOLS_VIEW` is perfect for me, note only one underline. The representation in the database should be irrelevant for their usage in code so let's make them consistent with python naming.